src/AutoRest.CSharp/Common/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -144,8 +144,7 @@ await foreach (var file in project.GetGeneratedFilesAsync())
                 {
                     // Ignore any errors while trying to output crash information
                 }
-                await AutoRestLogger.Fatal($"Internal error in AutoRest.CSharp{ErrorHelpers.FileIssueText}\nException: {e.Message}\n{e.StackTrace}");
-                return false;
+                throw;
```

Comment:
previously the exception stack trace somehow gets swallowed completely.
Throwing this out and this exception will be caught again where this is invoked and we will get full stack traces there.

Reply:
Is the context information sufficient? If not, we could `throw new XXXExceptio()` with more details.

Reply:
we actually do not want extra information here. If we wrap it, it will add an additional layer to the stack trace which is quite unnecessary